corehq/apps/geospatial/static/geospatial/js/geospatial_map.js

```diff
@@ -131,6 +133,28 @@ hqDefine("geospatial/js/geospatial_map", [
                 });
             });
 
+            self.setDisbursementParameters = function (parameters) {
+                var parametersList = [
+                    {name: gettext("Max cases per user"), value: parameters.max_cases_per_user},
+                    {name: gettext("Min cases per user"), value: parameters.min_cases_per_user}
+                ];
+
+                if (parameters.max_case_distance) {
+                    const maxCaseDistanceParamValue = `${parameters.max_case_distance} km`;
```

Comment:
nit: Km(s)?

Reply:
I far as I know (and am familiar with) it's always, say, `25 km` and not `25 kms`.